### DISCUSSION TITLE
Set search $PATH for Xcode on M1 install

### DISCUSSION THREAD

**Discussion by bcomnes** [Homebrew/discussions#2038] [Everyday usage]
I'm having issues getting homebrew programs to work nicely inside of Xcode. 

Example:

In my `~/.gitconfig`, I configure git to use `gpg` installed with homebrew:

```
[gpg]
	program = gpg
[commit]
	gpgsign = true 
```

```console
% which gpg 
/opt/homebrew/bin/gpg
```

This basically works fine in the CLI and many GUI programs, however, in Xcode, it does not. When I try to make a commit, it fails because it can't find `gpg` since, Xcode isn't searching `/usr/opt/homebrew/bin` by default. (It used to search `/usr/local/bin` by default). 

While I can supplement the $PATH in project Build settings, I have to do that for ever project, and the built in git client is broken because its not finding my homebrew tools. 

Any idea how I can update the macOS Default path for the entire Xcode program?

**Comment by SMillerDev** [comment]
This really seems like a question for the Apple forums.

**Reply by bcomnes** [reply]
Correctly setting the $PATH on account of the new M1 homebrew path is absolutely relevant to the usability of homebrew. If what once worked perfectly with XCode no longer works due to a missing assumption relating to the path changes in M1 homebrew, it is an issue that should be addressed within homebrew documentation at least. Or maybe it's just a bug with Xcode. Either way maybe someone here has already found a solution. 

But that's a good idea, I will ask in their developer forum as well, and cross post any discoveries I find.
